ts/src/krakenfutures.ts

```diff
@@ -1645,12 +1645,13 @@ export default class krakenfutures extends Exchange {
         //  'volumeQuote': 2.63}
         //
         const marketId = this.safeString (ticker, 'symbol');
+        const symbol = this.symbol (marketId);
         const timestamp = this.parse8601 (this.safeString (ticker, 'lastTime'));
         const fundingRate = this.safeNumber (ticker, 'fundingRate');
         const nextFundingRate = this.safeNumber (ticker, 'fundingRatePrediction');
         return {
             'info': ticker,
-            'symbol': marketId,
+            'symbol': symbol,
```

Comment:
I originally wrote it but it wasn't compatible with base exchange's fetchFundingRates implementation

Reply:
@youknowone I tested it and seems to be working fine
```
 p krakenfutures fetchFundingRate "BTC/USD:USD"   
Python v3.10.9
CCXT v3.1.34
krakenfutures.fetchFundingRate(BTC/USD:USD)
{'datetime': '2023-06-13T09:03:36.454Z',
 'estimatedSettlePrice': None,
 'fundingDatetime': None,
 'fundingRate': 0.48863570971999126,
 'fundingTimestamp': None,
 'indexPrice': None,
 'info': {'ask': '26210',
          'askSize': '0.395',
          'bid': '26206',
          'bidSize': '0.5037',
          'fundingRate': '0.488635709719991274',
          'fundingRatePrediction': '0.574588639840842062',
          'indexPrice': '26193.62',
          'last': '26205',
          'lastSize': '0.0001',
          'lastTime': '2023-06-13T09:03:36.454Z',
          'markPrice': '26208',
          'open24h': '25953',
          'openInterest': '1370.38900000000',
          'pair': 'XBT:USD',
          'postOnly': False,
          'suspended': False,
          'symbol': 'pf_xbtusd',
          'tag': 'perpetual',
          'vol24h': '2228.6211',
          'volumeQuote': '57829701.9162'},
 'interestRate': None,
 'markPrice': None,
 'nextFundingDatetime': None,
 'nextFundingRate': 0.5745886398408421,
 'nextFundingTimestamp': None,
 'previousFundingDatetime': None,
 'previousFundingRate': None,
 'previousFundingTimestamp': None,
 'symbol': 'BTC/USD:USD',
 'timestamp': 1686647016454}
```